axon-server/administration/cluster-template.md

```diff
@@ -0,0 +1,84 @@
+#Cluster template
+
+With the Cluster Template feature we enabled users to maintain cluster configuration as a code.
+
+Cluster template is defined as a YAML file which describes cluster configuration. It is possible to pre-define replication groups, contexts, metadata, applications (with tokens), users and their roles and that configuration can be shared across teams.
```

Comment:
```suggestion
The cluster template is defined as a YAML file, describing a cluster's configuration. It is possible to predefine replication groups, contexts, metadata, applications (with tokens), users and their roles, so that the configuration can be shared across teams.
```